src/internal/converters/eml/eml_test.go

```diff
@@ -27,11 +30,52 @@ func (suite *EMLUnitSuite) TestConvert_messageble_to_eml() {
 	body, err := os.ReadFile("testdata/email-with-attachments.json")
 	require.NoError(t, err, "reading test file")
 
+	out, err := FromJSON(body)
+	assert.NoError(t, err, "converting to eml")
+
 	msg, err := api.BytesToMessageable(body)
 	require.NoError(t, err, "creating message")
 
-	_, err = ToEml(msg)
-	// TODO(meain): add more tests on the generated content
-	// Cannot test output directly as it contains a random boundary
-	assert.NoError(t, err, "converting to eml")
+	assert.Contains(t, out, fmt.Sprintf("Subject: %s", ptr.Val(msg.GetSubject())))
+	assert.Contains(t, out, fmt.Sprintf("Date: %s", msg.GetSentDateTime().Format(time.RFC1123Z)))
+	assert.Contains(
+		t,
+		out,
+		fmt.Sprintf(
+			`From: "%s" <%s>`,
+			ptr.Val(msg.GetFrom().GetEmailAddress().GetName()),
+			ptr.Val(msg.GetFrom().GetEmailAddress().GetAddress())))
+
+	for _, addr := range msg.GetToRecipients() {
+		assert.Contains(
+			t,
+			out,
+			fmt.Sprintf(
+				`To: "%s" <%s>`,
+				ptr.Val(addr.GetEmailAddress().GetName()),
+				ptr.Val(addr.GetEmailAddress().GetAddress())))
+	}
+
+	for _, addr := range msg.GetCcRecipients() {
+		assert.Contains(
+			t,
+			out,
+			fmt.Sprintf(
+				`Cc: "%s" <%s>`,
+				ptr.Val(addr.GetEmailAddress().GetName()),
+				ptr.Val(addr.GetEmailAddress().GetAddress())))
+	}
+
+	for _, addr := range msg.GetBccRecipients() {
+		assert.Contains(
+			t,
+			out,
+			fmt.Sprintf(
+				`Bcc: "%s" <%s>`,
+				ptr.Val(addr.GetEmailAddress().GetName()),
+				ptr.Val(addr.GetEmailAddress().GetAddress())))
+	}
+
+	// Only fist 30 chars as it the data will be split
```

Comment:
nit: wording
```suggestion
	// Only fist 30 chars as the data will be split
```

Reply:
can you elaborate on this a bit more? I'm unsure what the nature of the "split" in this case is. Does it mean there's other output somewhere else we need to handle?

Reply:
It is possible for the tool to introduce newlines to limit the column width. I've update the comments as well.